### ISSUE TITLE
[lodash] Remove usage of _.omit

### ISSUE THREAD

**Issue by abdavid** [moleculerjs/moleculer#752]
See #433 for context.

**Describe the solution you'd like**
Remove usage of `_.omit` throughout Moleculer. 

**Describe alternatives you've considered**
Source: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_omit
```javascript
// Native
var { a, c, ...result2 } = object;
console.log(result2)
// output: { 'b': '2' }
```

**Additional context**
https://github.com/moleculerjs/moleculer/search?q=_.omit&unscoped_q=_.omit

**Comment by icebob** [comment]
For `omit` need a function because the name of properties comes from users.

**Comment by abdavid** [comment]
> For `omit` need a function because the name of properties comes from users.

So we are actually talking about https://lodash.com/docs/4.17.15#omitBy ?

**Comment by icebob** [comment]
No, it's the `_.omit` but the list or properties comes from another variable, so not static.

**Comment by abdavid** [comment]
> No, it's the `_.omit` but the list of properties comes from another variable, so not static.

This one here you are thinking about?
https://github.com/moleculerjs/moleculer/blob/07dbf1c0ead1038d157ec6616ac76509b6974549/src/service.js#L197

**Comment by icebob** [comment]
Yeah, e.g.

**Comment by abdavid** [comment]
We could do an inverse of suggestion from #748 then maybe?

```javascript
const omit = (o, xs) => {
    if(!Array.isArray(xs))
         xs = [xs];
    const objectPropertyNames = Object.getOwnPropertyNames(o)
    return xs.reduce((acc,x) => {
       if(objectPropertyNames.indexOf(x) > -1)
            acc[x] = o[x]
       return acc;       
    }, {})
}
```

**Comment by icebob** [comment]
By the way, `omit` & `pick` should work with nested paths as well :) `_.pick(obj, ["a.b", "a.c"]);`

**Comment by abdavid** [comment]
Noted :) I belive we then might be able to reuse #753 implementation there
